lib/timer.py

```diff
@@ -139,7 +140,19 @@ def run_jobs(cfg: UAConfig, current_time: datetime):
     Persist jobs-status with calculated next_run values to aid in timer
     state introspection for jobs which have not yet run.
     """
-    jobs_status_obj = timer_jobs_state_file.read()
+    jobs_status_obj = None
+    # If the file format is wrong we remove it, and after the jobs are
+    # executed it will be recreated with the proper data.
+    try:
+        jobs_status_obj = timer_jobs_state_file.read()
+    except InvalidFileFormatError:
+        try:
+            timer_jobs_state_file.delete()
+        except (OSError, PermissionError):
```

Comment:
I think we can just catch the generic Exception here and also log the exception just so we can better debug this if needed

Reply:
I think yes, logging the exception is good. But catching generic, well, I don't know. I prefer to be specific, people will complain about the "bad practice" again...